src/firefly/js/tables/ui/TableRenderer.js

```diff
@@ -220,14 +220,14 @@ function EnumSelect({col, tbl_id, filterInfoCls, onFilter}) {
 
 export function SelectableHeader ({checked, onSelectAll, showUnits, showTypes, showFilters, onFilterSelected, sx}) {
     return (
-        <Stack alignItems='center' height={1} justifyContent='end' py='1px' sx={sx}>
+        <Stack alignItems='center' height={1} justifyContent='space-between' py='2px' sx={sx}>
             <Checkbox size='sm'
                 tabIndex={-1}
                 checked={checked}
                 onChange={(e) => onSelectAll(e.target.checked)}/>
             {/*{showUnits && <Box height='1em'/>}*/}
             {/*{showTypes && <Box height='1em'/>}*/}
-            {showFilters && <FilterButton  iconButtonSize='32px'
+            {showFilters && <FilterButton  iconButtonSize='28px'
```

Comment:
Might consider changing this to rem units if it makes sense.

Reply:
Probably not.  The table library used is `fixed-data-table`.  Everything is in pixel size.